Tests/test.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+from dojo import Dojo
+
+
+class TestRoomCreation(unittest.TestCase):
+
+    """ Test cases to for successful room creation """
+
+    def setup(self):
+        self.dojo = Dojo()
+
+    def test_can_create_office(self):
+        self.assertEqual(len(self.dojo.offices), 0)
+        self.dojo.create_room('office', 'marylin')
+        self.assertEqual(len(self.dojo.offices), 1, 'An office has been successfully created!')
```

Comment:
@mnjaggah These tests were meant to test for success ... I'm working on tests to test for failure